src/domain/courseSearch/Search.tsx

```diff
@@ -311,13 +311,12 @@ const Search: React.FC<Props> = ({ scrollToResultList }) => {
                   minInputFixedValue={'18'}
                   maxInputValue={maxAgeInput}
                   maxInputLabel={t('courseSearch.search.ageLimitMax')}
-                  maxInputEndValue={'100'}
+                  maxInputEndValue={'99'}
```

Comment:
Just wondering why these values are originally strings and not numbers? Now this causes a lots of converting inside RangeDropDown component

Reply:
cuz hds textinput accepts string...